runit/kopano-smtpstd/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+source ../kopano-helper.sh
+
+ensure_config smtpstd.cfg
+ensure_dir /var/lib/kopano-smtpstd
+chown kopano:kopano /var/lib/kopano-smtpstd
+
+# read settings from smtpstd.cfg into env (emulates systemds EnvironmentFile)
+sed s/\ *=\ */=/g "/etc/kopano/smtpstd.cfg" > /tmp/smtpstd-env
+# shellcheck disable=SC2046
+export $(grep -v '^#' /tmp/smtpstd-env | xargs -d '\n')
+
+exec /sbin/setuser kopano /usr/sbin/kopano-smtpstd serve
```

Comment:
Please no user kopano. Use nobody or a specific system user for all services except the ones which actually need direct access to the kopano-server socket.